client/src/js/core/frame.js

```diff
@@ -16,14 +16,29 @@ Frame.prototype.motor = function(value) {
     // Multiplying by this value should make possible to write directly to PWM
     // Current range is 0 - 127 with first bit as direction
     // Gets 0-100 
-    let k = 1.27;
+    let k = 1.05;
```

Comment:
Does this make a difference? Did you find out `motor()` is called somewhere?

Reply:
it doesn't, but I changed it to stay consistent with `motors()`

Reply:
Maybe extract it out into a const and use it in motors as well?